### PR TITLE
Changes to the commands page

### PR DESCRIPTION
**Pull request by Miko0000**
Changes done to the commands page
- Updated looks
- Added a search function
- Added various animations
- Added a loading animation
- Added alternative way to set colors
- Added alternative way to set style specific to a screen size
- Added color changes
- Added jskit
- HTML structure is changed
- Indentation is changed to tab for the affected files

### CODE REVIEW

File: .gitmodules
Comment (Kile) [inline .gitmodules:1]: I don't love the idea of blindly depending on/running some code not owned by me from another repo. Do you mind instead copy pasting the relevant parts?

**Review by Kile — COMMENTED** [review]
Overall though very nice PR, thanks for helping out.

File: .gitmodules
Comment (Miko0000) [reply .gitmodules:1]: I changed it to a folder

File: flipsheets.css [hunk 1]
```diff
@@ -92,4 +94,23 @@ h2 {
...
+}
+
+#wrapper {
+    width: 100vw;
+    margin: 0px auto;
+    max-width: unset;
+}
+
+#thrc, .thrc {
+    justify-content: left;
```
Comment (Kile) [inline flipsheets.css]: It looks a bit weird to have them left aligned. Either align them to center or fill up the boxes dynamically so there is no large gap at the right
image

**Review by Miko0000 — COMMENTED** [review]
There's an issue with this commit.

**Comment by Kile** [comment]
What issue, works fine for me now?

**Comment by Miko0000** [comment]
Nevermind, It seems to be a glitch on my end and have fixed itself.

**Comment by Kile** [comment]
Awesome

**Review by Kile — APPROVED** [review]

**Pull request merged**